index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="app-bar">
+        <div class="logo">
+          <h1>My Logo</h1>
+        </div>
+        <ul class="nav">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <div class="menu-icon">
+          <i class="fas fa-bars purple-icon"></i>
+        </div>
+      </div>
+      <div class="content">
+        <div class="real-content">
+          <h1 class="h1">I'm David. Glad to see you!</h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <p class="link">LET'S CONNECT</p>
+          <div class="icons">
+            <i class="fa-brands fa-twitter fa-lg"></i>
+            <i class="fa-brands fa-linkedin fa-lg"></i>
+            <i class="fa-brands fa-medium fa-lg"></i>
+            <i class="fa-brands fa-github fa-lg"></i>
+            <i class="fa-regular fa-hand-peace fa-lg"></i>
+          </div>

```

Comment:
- [ ] As the previous reviewer suggested, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly apply the changes that the previous reviewer requested.